content/attacks/posts/2022-01-08-LCX.md

```diff
@@ -0,0 +1,46 @@
+---
+date: 2022-01-08
+target-entities: LCX
+entity-types:
+  - Exchange
+  - Custodian
+attack-types: Wallet Hack
```

Comment:
Please, add loss header in all your submission as below:
`loss: 8000000`
The loss is USD value of stolen assets in the moment of hack